rules/license.bzl

```diff
@@ -107,11 +110,12 @@ def license(
             fail("Can not use both license_kind and license_kinds")
         license_kinds = [license_kind]
 
-    # Make sure the file exists as named in the rule. A glob expression that
-    # expands to the name of the file is not acceptable.
-    srcs = native.glob([license_text])
-    if len(srcs) != 1 or srcs[0] != license_text:
-        fail("Specified license file doesn't exist: %s" % license_text)
+    if _require_license_text_is_a_file:
```

Comment:
do you plan to remove this at some point? If so, maybe add a comment near the definition of _require_license_text_is_a_file to explain when. If not, just remove this section.

Reply:
When we import to google, the copybara transform will turn it back on.
I added the comment above about what it is for.